src/app/eth-memento/module/component/TxsGridFields.ts

```diff
@@ -0,0 +1,124 @@
+import { ITranslation } from "plugin-api/ITranslation";
+import { GridFields } from "@alethio/ui/lib/control/grid/state/GridFields";
+import { EthRenderer } from "@alethio/ui/lib/data/gridRenderer/EthRenderer";
+import { AccountHashRenderer } from "@alethio/explorer-ui/lib/grid/dataRenderer/account/AccountHashRenderer";
+import { BlockLinkRenderer } from "@alethio/explorer-ui/lib/grid/dataRenderer/block/BlockLinkRenderer";
+import { DateTimeRenderer } from "@alethio/ui/lib/data/gridRenderer/DateTimeRenderer";
+import { TxTypeExtraRenderer } from "./txsGrid/TxTypeExtraRenderer";
+import { TransferDirectionRenderer } from "./txsGrid/TransferDirectionRenderer";
+import { ITxLiteByAccount } from "app/eth-memento/data/tx/byAccount/ITxLiteByAccount";
+import { TxLinkRenderer } from "@alethio/explorer-ui/lib/grid/dataRenderer/tx/TxLinkRenderer";
+
+enum ITxsGridFieldKeys {
+    Type = "type",
+    Hash = "hash",
+    From = "from",
+    Direction = "direction",
+    To = "to",
+    Value = "value",
+    Fee = "fee",
+    BlockNo = "blockNumber",
+    BlockTime = "blockTime"
+}
+
+export class TxsGridFields extends GridFields<ITxLiteByAccount> {
+    constructor(t: ITranslation, locale: string, ethSymbol: string, currentAccountAddress: string) {
+        super();
+        this.fields = [{
```

Comment:
It's not really a subset. Although many fields are identical, some are slightly different. For extended, the status bubble is added to type column, for memento, since the type is missing, the status bubble is added to the hash